bc_obps/registration/migrations/0032_delete_reporting_activity_model.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.7 on 2024-08-13 17:32
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0031_activity_model'),
+        ('reporting', '0004_V1_6_0')
+    ]
+
+    operations = [
+      migrations.DeleteModel(name='ReportingActivity'),
+      migrations.DeleteModel(
+            name='HistoricalReportingActivity',
+        ),
```

Comment:
nit: We could add this to the end of the previous migration file, just to have one less file to maintain.